contrib/lang/scala/packages.el

```diff
@@ -47,6 +47,78 @@ which require an initialization must be listed explicitly in the list.")
         (kbd "n") 'forward-button
         (kbd "N") 'backward-button)
 
+      (defun ensime-gen-and-reload()
+        (interactive)
+        (progn
+          (sbt-command "gen-ensime")
+          (ensime-shutdown)
+          (ensime))
+        )
+      
+      (evil-leader/set-key-for-mode 'scala-mode
+        "mg"     'ensime-edit-definition
+
+        "m."      'ensime-gen-and-reload
+        "m,"      'ensime
```

Comment:
Using `, ,` and `, .` for ensime-server commands. Could probably be moved to some more mnemonic binding.